sys-fs/cryfs/cryfs-0.9.7.ebuild

```diff
@@ -36,10 +36,20 @@ RDEPEND=">=dev-libs/boost-1.56:=
 DEPEND="${RDEPEND}
 	${PYTHON_DEPS}"
 
+src_prepare() {
+	# remove tests that require internet access to comply with Gentoo policy
+	sed -i '/CurlHttpClientTest.cpp/d' test/cpp-utils/CMakeLists.txt || die
+	sed -i '/FakeHttpClientTest.cpp/d' test/cpp-utils/CMakeLists.txt || die
```

Comment:
Minor nitpick but you can pass multiple `-e` expressions to sed, i.e. `sed -i -e /foo/d -e /bar/d ...`